### PR TITLE
Add `THANKS` callout

### PR DESCRIPTION
**Pull request by straight-shoota**
Formats all thank-you messages to our sponsors in the release notes as a callout.
So far this uses the standard callout template, but we could also go for something fancier in the future.
There's als a CTA button to invite new sponsors.

grafik

### CODE REVIEW

File: _releases/2016-05-05-crystal-0.16.0-released.md [hunk 1]
```diff
@@ -261,6 +261,7 @@ Make sure to read the [changelog](https://github.com/crystal-lang/crystal/releas
 
 ## Thank you!
```
Comment (beta-ziliani) [inline _releases/2016-05-05-crystal-0.16.0-released.md:263]:
```suggestion
```

File: _releases/2014-12-31-crystal-0.5.6-release.md [hunk 2]
```diff
@@ -52,7 +52,8 @@ and people won't have to worry about which documentation generator to choose. We
...
 
-Now there's just one thing missing: the documentation itself! We will add it, little by little, and you
-are very welcome to help us out. Luckily [Yukihiro Matsumoto kindly let us use some Ruby documentation as
-a starting point](https://twitter.com/yukihiro_matz/status/549317901002342400), given that a lot of
-APIs are very similar, if not equal. So どうもありがとうございます to him :-)
+> **THANKS:**
+> Now there's just one thing missing: the documentation itself! We will add it, little by little, and you
+> are very welcome to help us out. Luckily [Yukihiro Matsumoto kindly let us use some Ruby documentation as
+> a starting point](https://twitter.com/yukihiro_matz/status/549317901002342400), given that a lot of
+> APIs are very similar, if not equal. So どうもありがとうございます to him :-)
```
Comment (beta-ziliani) [inline _releases/2014-12-31-crystal-0.5.6-release.md:59]: I don't think this callout makes sense here. This is specific to the documentation and a call to action more than an actual thanking.

```suggestion
Now there's just one thing missing: the documentation itself! We will add it, little by little, and you
are very welcome to help us out. Luckily [Yukihiro Matsumoto kindly let us use some Ruby documentation as
a starting point](https://twitter.com/yukihiro_matz/status/549317901002342400), given that a lot of
APIs are very similar, if not equal. So どうもありがとうございます to him :-)
```

**Review by beta-ziliani — APPROVED** [review]
Really nice! Just a few fixes left.

File: _releases/2014-12-31-crystal-0.5.6-release.md (hunk 2 above)
Comment (straight-shoota) [reply _releases/2014-12-31-crystal-0.5.6-release.md:59]: Good catch! 😅

**Review by beta-ziliani — APPROVED** [review]

**Pull request merged**